CONTRIBUTING.md

```diff
@@ -7,6 +7,12 @@ Facebook has adopted a Code of Conduct that we expect project participants to ad
 ## Our Development Process
 Most of our work will be done in public directly on GitHub. There may be changes done through our internal source control, but it will be rare and only as needed.
 
+## Setup
+1. `git clone` the project
+2. update submodules by running `git submodule update --init` in the directory
```

Comment:
Also, technically - it's updating submodules, so can we replace `update` with `Fetch required submodules`